.travis.yml

```diff
@@ -5,3 +5,5 @@ node_js:
   - "0.12"
   - "0.10"
 sudo: false
+after_success:
+  - bash <(curl -s https://codecov.io/bash)
```

Comment:
For posterity's sake, we decided to leave this 3rd party script as-is.  If codecov were to be hacked, the attack surface would be limited to leaking access keys to our open source SauceLabs account for some free test automation.  If that happened we could rotate the keys.